echopype/echodata/sensor_ep_version_mapping/v05x_to_v06x.py

```diff
@@ -700,7 +700,10 @@ def _rearrange_azfp_attrs_vars(ed_obj, sensor):
             "DS",
             "EL",
             "TVR",
-            "VTX",
+            "VTX0",
+            "VTX1",
+            "VTX2",
+            "VTX3",
```

Comment:
Actually, leave this module as is, w/o changes. That's because it's intended only to address the conversion of files from a very old version of echopype (0.5) to another old version (0.6)
```suggestion
            "VTX",
```